Add docs for Scattered bid adapter

almendar opened the pull request:
<!--
Thanks for improving the documentation 😃
Please give a short description and check the matching checkboxes to help us review this as quick as possible.
-->

## 🏷 Type of documentation
<!-- Remove items that don't apply and/or select an item by changing [ ] to [x] -->
- [X] new bid adapter

## 📋 Checklist
- [x] For new adapters check [submitting your adapter docs](https://docs.prebid.org/dev-docs/bidder-adaptor.html#submitting-your-adapter)

jeanstemp commented:
@almendar, can you add one more line to the metadata in the header? We updated the navigation and all bidder docs now need to include `sidebarType: 1` with the header data.
cc @bretg

almendar commented:
Sure I'll update it. Sorry for delay I was dragged by other task from this.

almendar commented:
@jeanstemp I've added the needed info. Also In the mean time we obtained our own GLV ID so I've also included it. 

Sibling PR was also updated with it https://github.com/prebid/Prebid.js/pull/9295


@fowler446 Can we do the second round? :)

almendar commented:
Hey, can we proceed with the docs? :)

The adapter code was merged :)

bretg commented:
We don't merge docs until after the Prebid.js version containing the adapter actually goes out. You've been merged, but not yet released.
